r-package/grf/bindings/RegressionForestBindings.cpp

```diff
@@ -60,16 +60,14 @@ Rcpp::List regression_train(Rcpp::NumericMatrix train_matrix,
       data->set_weight_index(sample_weight_index - 1);
   }
   
+  ForestOptions options(num_trees, ci_group_size, sample_fraction, mtry, min_node_size, honesty,
+                        honesty_fraction, honesty_prune_leaves, alpha, imbalance_penalty, num_threads, seed, clusters, samples_per_cluster);
   if(mbb){
       ForestOptions options(num_trees, ci_group_size, sample_fraction, mtry, min_node_size, honesty,
           honesty_fraction, honesty_prune_leaves, alpha, imbalance_penalty, num_threads, seed, clusters, samples_per_cluster,
           blocklength, blocknum);
   }
```

Comment:
I just want to make sure about these lines. Do they mean the different options depending on the T/F of mbb? Could you please have a quick check?
When I write if.. else..., there is an error saying the "options" is undeclared. So I write the option of mbb part like this.